EdFi.Ods.Utilities.Migration/logistics/scripts/modules/build-edfi-standard-descriptors.psm1

```diff
@@ -0,0 +1,148 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+$ErrorActionPreference = "Stop"
+$repositoryNames = @('EdFi.Ods.Utilities.Migration')
+& "$PSScriptRoot\..\..\..\logistics\scripts\modules\load-path-resolver.ps1"  $repositoryNames 
```

Comment:
i am not sure we should keep logistics in the path. should we consider scripts off the root instead?

Reply:
is this path resolver referencing the ods project?

Reply:
we have already scripts folder  for migration ,So it kept with logistics folder to separate it out .

Reply:
I moved to EdFi.Ods.Utilities.Migration\EdFi.Standard.Descriptors folder